src/service/performance-impl.js

```diff
@@ -715,7 +715,7 @@ export class Performance {
    *     this directly.
    */
   tick(label, opt_delta) {
-    const value = opt_delta == undefined ? this.win.Date.now() : undefined;
+    const value = opt_delta == undefined ? this.now() : undefined;
```

Comment:
Can this be introduced in a backward-compatible way for existing AMP viewers that rely on the values being an absolute timestamp, as opposed to a time delta in the performance clock (relative to `timeOrigin`)?

IIUC, this change completely removes the current absolute timestamp value and replaces it with the performance clock value. Gmail's AMP viewer is currently relying on the value being absolute, so changing it to be relative would break Gmail's metrics. We raised the same concern in the reverted PR [here](https://github.com/ampproject/amphtml/pull/16119#issuecomment-399205526), but we never got a response.

One way to introduce this in a backward-compatible way is to send both the absolute timestamp using `Date.now()` and the relative delta using the performance clock in the message posted to the viewer, in different properties of the message payload.

Reply:
Thank you for catching this! I was holding off on merging because I hadn't yet found the reason why the last one was reverted :). We can use `performance.timeOrigin + performance.now()` to keep the absolute timestamp.

Reply:
I'm going to wait to update this further until I land both https://github.com/ampproject/amphtml/pull/26568 and https://github.com/ampproject/amphtml/pull/26567.

Reply:
👏👏👏

Let's just switch this back to `Date.now()`, no need to be any more precise with it.

Reply:
Eh, `performance.now()` is still better than Date because of the clock skew issues right?

Reply:
Yeah, it's also us who requested for using the performance clock because our metrics currently suffer from clock skew. :P converting the relative timestamp in the performance clock to an absolute one SGTM since that should be backward compatible.